### ISSUE TITLE
Feature Request: Add model preset reference to .materialtype as model preset preference for Materia Editor.

### ISSUE THREAD

**Issue by galibzon** [o3de/o3de#9054]
**Is your feature request related to a problem? Please describe.**
This issue comes from: https://github.com/o3de/o3de/issues/8719

**Describe the solution you'd like**
Implement the concept of an optional default model for material types

- Add a setting to indicate that the Eye.materialtype should use the Eye model by default. This should be a new settings registry item.

- When the Material Editor opens a material, it should automatically switch to the indicated preview model.

- When the Material Editor switches to a material that doesn't have a default model, it should use whatever model was active before the most recent default model was displayed (i.e. you are working with a StandardPbr material with the cylinder model, then open an eye material and it switches to the eye model automatically, then you switch back to the StandardPbr material and it uses the cylinder again. Then you open another StandardPbr material and it's still using the cylinder).

- There should be a checkbox or something to disable automatically switching to default models.

**Describe alternatives you've considered**
N/A

**Additional context**
See issue:  https://github.com/o3de/o3de/issues/8719

**Comment by galibzon** [comment]
@santorac to review possibility of adding the feature to .materialtype. If successful, @gadams3 can follow up with using the model preset preference that comes from .materialtype.

**Comment by santorac** [comment]
I don't recommend adding this to the .materialtype file. Instead I think it should go in the settings registry, as the ticket description indicates. This will allow game teams to customize the default model for a material type without having to modify the material type itself, which might be stored in a shared gem used by other projects. But if it's discovered that it makes sense to add to .materialtype, I think that's something that would be easy for @gadams3 to do himself (just a new "DefaultPreviewModel" setting with a model preset filename). We could even have both, where the registry can override what's in the .materialtype.

**Comment by gadams3** [comment]
I agree that adding this to the settings registry would be the most flexible at least invasive option. There's no major plumbing that needs to be done. Just adding the logic to check the settings registry for a a default model for a given material type and selecting it whenever a material using that type is opened.

**Comment by gadams3** [comment]
I don't think it's necessary to go back to a previously selected model.
